src/main/modules/session/index.ts

```diff
@@ -48,3 +49,7 @@ export class SessionStorage {
       });
   }
 }
+
+export type ReqWithSession = Request & {
```

Comment:
i added something in my pr for this, tho you way seems cleanier

`declare module 'express-session' {
  export interface SessionData {
    user: Record<string, unknown>
  }
}
`